### ISSUE TITLE
Keyboard not responding as it should on rhel8

### ISSUE THREAD

**Issue by codedbyanne** [norment/tsd_issues#74]
To begin with I could type in a terminal window to ssh onto the appnode. From the appnode I started rstudio. Here completely the wrong things happen when I press short-cut keys.
To test whether this was an rstudio issue or general I started a new terminal window and nano from the login rhel8 node. The short cut keys did not work here either.
After that nothing happens when I type in the terminal window any more. 

So basically not possible to work on rhel8...

**Comment by ofrei** [comment]
Hi @annekristin Thank you for reporting this - very important. Could you also submit an RT ticket, and paste here the number? Me and @Sandeek will follow up on this with USIT TSD team. 

Also, please clarify if you use Horizon VMWare client stand alone program, or a connection via browser? Could you please try the other option and let us know if it works or not?

**Comment by codedbyanne** [comment]
@ofrei Do I submit an RT ticket by sending an e.mail to tsd-drift@usit.uio.no or is that something else?

I used Horizon VMWare connection via Firefox browser. The standalone client does not start on my laptop, even though it said it installed without problems..

Anne-Kristin

**Comment by Sandeek** [comment]
@annekristin  Please submit an RT ticket, it will be easier to follow up with TSD. yes, email to tsd-drift@usit.uio.no

**Comment by Sandeek** [comment]
@annekristin as suggested by Martin, please try logging in with VMware horizon client as there are keyboard layout mapping issues in the browser

**Comment by codedbyanne** [comment]
Thank you for suggesting that. Unfortunately the client doesn't work in my case. It installed without problems, but will not start.. There are no error messages, it simply doesn't start. I am in the process of changing to a new laptop, so will test again there, hopefully soon. Anne- Kristin23. mar. 2021 16:12 skrev Sandeep Karthikeyan ***@***.***>:
@annekristin as suggested by Martin, please try logging in with VMware horizon client as there are keyboard layout mapping issues in the browser

**Comment by ofrei** [comment]
@annekristin Thank you. To clarify, these are two separate issues that we discuss here
1. Keyboard layout problems in rhel8 when accessed via browser
2. Problems with installing & running VMWare horizon client

For the first issue, please e-mail to tsd-drift@usit.uio.no and copy here the RT ticket number .

For the second issue - what is your operating system? For me VMware horizon runs well on  Ubuntu and Windows 10 (and the remote connection experience is much better than with browser access).
